package.json

```diff
@@ -20,11 +20,9 @@
   "main": "src/index.js",
   "browser": {
     "./src/repo/lock.js": "./src/repo/lock-memory.js",
+    "./src/repo/default-root-options.js": "./src/repo/ldefault-root-options.browser.js",
     "datastore-fs": "datastore-level"
   },
-  "bin": {
-    "jsipfs-migrations": "./src/cli.js"
-  },
```

Comment:
This is meant to be a patch, but this is going to break cli usage.

Reply:
TBH I'm not sure why there's a cli component here at all.  IPFS doesn't use it, it only seem to be there to make the sharness tests pass.

Reply:
ok, seems reasonable to drop this then.

Reply:
You could, I suppose, use it to do a repo migration outside of IPFS, but if you use a non-standard config it'll break because there's no way to pass detailed config in.  Better to just let IPFS take care of it.